lib/ui/painting/image_decoder.cc

```diff
@@ -4,13 +4,35 @@
 
 #include "flutter/lib/ui/painting/image_decoder.h"
 
-#include <algorithm>
```

Comment:
I think we added these explicit includes of `<algorithm>` for the g3 build. Please verify that they went along with the code that got shuffled around.

Reply:
I removed all existing headers and tried to include what was being used as there were a few spots where there were spurious includes. But you're right, std::max would need to include `algorithm` but in the _skia TU. I'll add it there.